test/integration/esClient.js

```diff
@@ -0,0 +1,64 @@
+const chai = require('chai');
+const esClient = require('../../app/lib/esClient');
+const utils = require('./testUtils');
+
+const expect = chai.expect;
+
+describe('esClient', function test() {
+  this.timeout(utils.maxWaitTimeMs);
+
+  before((done) => {
+    utils.waitForSiteReady(done);
+  });
+
+  describe('getQuery', () => {
+    const location = { longitude: -1.46519099452929, latitude: 54.0095586395326 };
+    it('should return pharmacies as an array', (done) => {
+      esClient.getPharmacies(location).then((pharmacies) => {
+        expect(pharmacies).to.be.an('array');
+        expect(pharmacies.length).to.be.greaterThan(0);
+        expect(pharmacies[0].name).to.exist;
+        done();
+      }).catch(done);
+    });
+
+    it('number of results should default to 2500', (done) => {
```

Comment:
This test doesn't really do what it says it does. I'd change this to inspect the query created and make sure the number of results requested was correct. Obviously that assumes the underlying technology the request is being made to works but we shouldn't be testing that anyway.